functional-samples/ai.gemini-on-device-alt-texter/README.md

```diff
@@ -0,0 +1,13 @@
+# On-device multimodal AI with Gemini Nano - image understanding
+
+This sample demonstrates how to use the image understanding capabilities of the multi-modal Gemini Nano API preview. To learn more about the API and how to sign-up for the origin trial, head over to [Built-in AI on developer.chrome.com](https://developer.chrome.com/docs/extensions/ai/prompt-api).
+
+## Overview
+
+This extension adds a context menu entry for images on the web to generate an alt text description that is displayed in a popup window.
```

Comment:
Can we briefly mention that this also uses the translator API to provide translations?

Reply:
done